src/components/Form.js

```diff
@@ -17,21 +18,21 @@ Form.propTypes = {
 
 export default function Form({ questions, onSaveHouse }) {
   const [formData, setFormData] = useState({})
-  const [currentStep, setCurrentStep] = useState(0)
+
+  const { currentStep, advanceStep } = useAdvanceStep()
 
   const totalSteps = questions.length - 1
   const { question, options } = questions[currentStep]
 
   return (
     <FormStyled onSubmit={handleSubmit}>
```

Comment:
[According to this article on MDN ](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/form_role#description)a form needs an accessible name to be recognized as a landmark. Maybe you can add a heading and reference to it with `aria-labelledby` or you can use `aria-label`.

Reply:
I wanted to add a bit more description later on in form of a landing page. But will add an `aria-label` 👍